app/components/modal/InstructionModal/InstructionModal.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from 'react'
+import Image from 'next/image'
+import {
+  Bs1Circle,
+  Bs2Circle,
+  Bs3Circle,
+  Bs4Circle,
+  Bs5Circle,
+  Bs6Circle,
+} from 'react-icons/bs'
+import styles from './InstructionModal.module.css'
+
+const steps = [
+  {
+    step: 1,
+    title: 'ChatGPT APIキーの取得方法',
+    subtitle: 'Open AI社のアカウントを作成・ログインする',
+    description:
+      'Open AI社のAPIページにアクセスし、「Start building」をクリックしてください。',
+    image: '/SecondModalview.png',
+    alt: 'Open AIのアカウント作成画面',
+    icon: <Bs1Circle className={styles.icon} />, // ステップ1のアイコン
```

Comment:
Figmaで使用しているアイコンにしてほしいです！（6番まで）
```suggestion
    icon: <TbCircleNumber1Filled className={styles.icon} />, // ステップ1のアイコン
```